loki/README.md

```diff
@@ -144,6 +144,30 @@ kubectl -n ${KYMA_LOKI_EXAMPLE_NS} get pod -l app=loki,release=${HELM_RELEASE_NA
      'time={NANOSECONDS}'
    ```
 
+### Access Grafana
+
+To access Grafana, either use kubectl port forwarding, or expose it using the Kyma Ingress Gateway.
+
+* To access Grafana using port forwarding, run:
```

Comment:
The access with port-forwarding is already mentioned above. Maybe we should just move the APIRule based exposure also to the "Installation with Grafana" section.